fossunited/fossunited/utils.py

```diff
@@ -13,7 +13,98 @@ def formatted_datetime_with_tz(datetime_string):
 	return datetime_string
 
 
+def format_time_with_zone(time, format_string="%-I:%m %p %Z"):
+	return format_time(time, format_string)
+
+
 @frappe.whitelist()
 def update_rsvp_count(rsvp):
 	count = frappe.db.count("FOSS Event RSVP Submission", {"linked_rsvp": rsvp})
 	frappe.db.set_value("FOSS Event RSVP", rsvp, "rsvp_count", count)
+
+
+def is_session_user_team_member(chapter):
+	members = frappe.get_doc("FOSS Chapter", chapter).chapter_member
```

Comment:
```suggestion
	members = frappe.get_doc("FOSS Chapter", chapter).chapter_members
```

The field name for child tables (table fields) should be plural...